src/pt/BreadCrumbs.jl

```diff
@@ -0,0 +1,62 @@
+"""
+A struct that provides a basic, user-friendly interface to Pigeons. Only two inputs
+are required, in positional order:
+$FIELDS
+
+!!! note
+
+    The PT state is initialized using a random sample from the reference.  
+"""
+struct BreadCrumbs{TRefDist <: Distributions.Distribution, TTarget}
+    """A function that evaluates the target log potential"""
+    target_log_potential::TTarget
+    """A Distributions.jl distribution used as reference"""
+    reference_distribution::TRefDist
+end
+
+# Target for a BreadCrumbs input
+struct BreadCrumbsTarget{TBC <: BreadCrumbs}
```

Comment:
This seems superfluous?

Reply:
I.e. instead can we write the dispatches on Distributions types directly?

Reply:
Ah! I think you're right! At least for the reference. I don't think people are thinking of passing the loglikelihood as a Distribution on the data given the parameter. That would also be very restrictive given that Distributions.jl only has simple models.